### ISSUE TITLE
audit warning

### ISSUE THREAD

**Issue by vext01** [bheisler/criterion.rs#650]
```
$ cargo audit -Dwarnings
    Fetching advisory database from `https://github.com/RustSec/advisory-db.git`
      Loaded 499 security advisories (from /home/vext01/.cargo/advisory-db)
    Updating crates.io index
    Scanning Cargo.lock for vulnerabilities (137 crate dependencies)
Crate:     atty
Version:   0.2.14
Warning:   unsound
Title:     Potential unaligned read
Date:      2021-07-04
ID:        RUSTSEC-2021-0145
URL:       https://rustsec.org/advisories/RUSTSEC-2021-0145
Dependency tree:
atty 0.2.14
└── criterion 0.4.0

error: 1 denied warning found!
```

No newer version of atty at this time though...

**Comment by tilosp** [comment]
Duplicate of #629

**Comment by waywardmonkeys** [comment]
This can be closed. It was fixed in #628.
